packages/react/src/file-upload/FileUploadFile.tsx

```diff
@@ -26,41 +25,35 @@ export const FileUploadFile = ({
 	const imagePreview = useMemo(() => getImageThumbnail(file), [file]);
 	return (
 		<Flex
-			rounded
 			as="li"
-			aria-label={`${status === 'success' ? 'Uploaded file' : 'File'}. ${name}`}
+			background={status === 'success' ? status : 'shade'}
 			gap={0.5}
 			justifyContent="space-between"
-			css={{ backgroundColor: TONE_MAP[status] }}
+			rounded
 		>
 			<Flex>
 				{showThumbnail && <FileUploadFileThumbnail src={imagePreview} />}
-				<Flex alignItems="center" paddingLeft={1} gap={0.5}>
+				<Flex alignItems="center" gap={0.5} paddingLeft={1}>
 					{status == 'success' && (
 						<Box flexShrink={0}>
 							<SuccessFilledIcon
-								color="success"
-								size="md"
 								aria-hidden="false"
 								aria-label="Success"
+								color="success"
 								css={{ display: 'block' }}
+								size="md"
 							/>
 						</Box>
 					)}
 					{href ? (
-						<Text paddingY={1.5}>
-							<TextLink
-								href={href}
-								target="_blank"
-								rel="noopener noreferrer"
-								css={{ wordBreak: 'break-all' }}
-							>
+						<Text breakWords paddingY={1.5}>
+							<TextLink href={href} rel="noopener noreferrer" target="_blank">
```

Comment:
Same comment about `target="_blank"` links and hidden text. If this is something you think we can do, maybe this should be incorporated into our link components (DefaultLinkComponent, TextLink etc) in a separate PR? `if (props.target === '_blank' && props.html) ...` or something similar?